scripts/test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+./coins/monero/c/monero/build/release/bin/monerod --detach --regtest --offline --fixed-difficulty=1 --rpc-bind-ip 127.0.0.1 --rpc-bind-port 18081
+cargo test
```

Comment:
`--all-features` ?

Reply:
Looks like this isn't feasible because try-runtime doesn't work for the current Substrate runtime. One error seems to be a macro not realizing what it's doing, the others... I'm not sure about. We could remove try-runtime for now?